EXPLAINER.md

```diff
@@ -130,6 +130,9 @@ The browser may omit a `fullscreenchange` event prior to any scripts being execu
 would need to account for this behavior in fullscreen-popup scenarios, or browsers would need to carefully ensure that `fullscreenchange` is not emitted until after
 scripts are ready to handle the event.
 
+#### **Delays entering fullscreen**
+There may exist a delay (unintentional or malicious) between the time the popup is created during the original [user activation](https://html.spec.whatwg.org/multipage/interaction.html#tracking-user-activation) and the time that the [document.documentElement](https://dom.spec.whatwg.org/#ref-for-dom-document-documentelement%E2%91%A0) becomes available. For example a slow network or intentional throttling by a malicious server may cause a significant delay which would make the popup become fullscreen long after user input. In such case, the user agent would display an empty or loading page until the [document.documentElement](https://dom.spec.whatwg.org/#ref-for-dom-document-documentelement%E2%91%A0) is available and becomes fullscreen. Therefore a web application has no control over the content shown in the popup between the user gesture and the time which the document element is available and made fullscreen.
```

Comment:
Maybe link to the document element concept instead? https://dom.spec.whatwg.org/#document-element

Reply:
Consider allowing user agents to abort the fullscreen steps in these cases, maybe:

> Slow network conditions or intentional throttling by a malicious server may introduce a significant delay between the popup window opening and the popup's [document element](https://dom.spec.whatwg.org/#ref-for-dom-document-documentelement%E2%91%A0) becoming available to fullscreen. In such case, the user agent could postpone or abort the steps for making the popup's document element fullscreen.

I also offer a rewrite, but might suggest just omitting the last sentence:

> Web applications generally have limited control over the appearance of windows before their content is loaded.

Reply:
Done. Thanks for reviewing!